### PR TITLE
Cosmos info validator

### PR DESCRIPTION
**Pull request by kaladinlight**

### CODE REVIEW

File: go/coinstacks/cosmos/api/api.go [hunk 1]
```diff
@@ -314,3 +307,52 @@ func (a *API) EstimateGas(w http.ResponseWriter, r *http.Request) {
...
+	v := cosmos.Validators{
+		Validators: validators,
+	}
+
+	api.HandleResponse(w, http.StatusOK, v)
+}
+
+// swagger:route Get /api/v1/validators/{pubkey} v1 GetValidator
+//
+// Get the list of current validators
```
Comment (kaladinlight) [inline go/coinstacks/cosmos/api/api.go:334]: Update commment for single validator

File: go/coinstacks/cosmos/api/handlers.go [hunk 2]
```diff
@@ -77,62 +78,122 @@ func (h *Handler) StartWebsocket() error {
...
+	}
+
+	bBondedTokens, _, err := new(big.Float).Parse(bondedTokens, 10)
+	if err != nil {
+		return nil, err
+	}
+
+	bInflationRate := new(big.Float).Quo(bAnnualProvisions, bTotalSupply)
+	bBondedTokenRatio := new(big.Float).Quo(bBondedTokens, bTotalSupply)
+	bTaxRate := new(big.Float).Mul(bInflationRate, (new(big.Float).Sub(big.NewFloat(1), bCommunityTax)))
```
Comment (kaladinlight) [inline go/coinstacks/cosmos/api/handlers.go:123]: rewardRate

File: go/coinstacks/cosmos/api/handlers.go [hunk 3]
```diff
@@ -77,62 +78,122 @@ func (h *Handler) StartWebsocket() error {
...
+	if err != nil {
+		return nil, err
+	}
+
+	bBondedTokens, _, err := new(big.Float).Parse(bondedTokens, 10)
+	if err != nil {
+		return nil, err
+	}
+
+	bInflationRate := new(big.Float).Quo(bAnnualProvisions, bTotalSupply)
```
Comment (kaladinlight) [inline go/coinstacks/cosmos/api/handlers.go:121]: easy understand comment showing math

File: go/pkg/cosmos/bank.go [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+	_, err := c.cosmos.R().SetResult(&res).Get("/cosmos/distribution/v1beta1/params")
+	if err != nil {
+		return "0", errors.Wrap(err, "failed to get community tax")
+	}
+
+	return res.Params.CommunityTax, nil
+}
+
+func (c *HTTPClient) GetBondedTokens() (string, error) {
```
Comment (kaladinlight) [inline go/pkg/cosmos/bank.go:56]: grpc endpoint

**Review by elmutt — APPROVED** [review]

**Review by theoboldfrazier — APPROVED** [review]

**Pull request merged**